package.json

```diff
@@ -23,6 +23,15 @@
     "profile": "beefy tests/bench/bench.js --open -- -t brfs -t ./tests/bench/xform-rules.js -r espree",
     "coveralls": "cat ./coverage/lcov.info | coveralls"
   },
+  "gitHooks": {
+    "pre-commit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
```

Comment:
I'm happy if it runs markdown lint as well because I have sometimes forgotten it.

Reply:
Good point, me too.